packages/scss/src/components/_nav-side.scss

```diff
@@ -41,27 +50,56 @@
 
 .navSide-scrollWrapper {
 	width: _component("navSide.width");
-	padding: _theme("spacings.smaller");
+	padding: 0.5em;
+	margin: 0;
+    list-style-type: none;
 }
 
-.navSide-item + .navSide-item, .navSide-item-subMenu-link + .navSide-item-subMenu-link {
-	margin-top: 2px;
+.navSide-item:not(.mod-mobileToggle) + .navSide-item, .navSide-item-subMenu-link + .navSide-item-subMenu-link,
+.navSide-item-subMenu-item + .navSide-item-subMenu-item {
+	margin-top: .125em;
 }
 
+
 // MOBILE TOGGLE
 .navSide-item.mod-mobileToggle {
 	display: none;
 }
 
 .navSide-item-link, .navSide-item-subMenu-link {
+	background-color: transparent;
+	border: 0;
+	cursor: pointer;
+	text-align: left;
+	width: 100%;
+	height: 100%;
 	align-items: center;
 	color: _component("navSide.fullwidth-palette.text");
 	display: flex;
 	text-decoration: none;
 	transition: all _theme("commons.animations.durations.fast") ease;
 	user-select: none;
-	height: 100%;
 	border-radius: _theme("commons.border.radius");
+
+	&[aria-expanded="true"] {
```

Comment:
code redondant avec la ligne 207 de l'état is-open

Reply:
Not for the mobile button and for the future.